apps/teams-test-app/src/components/AppInitialization.tsx

```diff
@@ -45,12 +45,35 @@ const NotifyFailure = (): React.ReactElement =>
     },
   });
 
+const NotifyExpectedFailure = (): React.ReactElement =>
+  ApiWithTextInput<app.ExpectedFailureReason>({
+    name: 'appInitializationExpectedFailure2',
```

Comment:
nit : '2', We can get rid of 2 since its the first time we are writing tests for this

Reply:
will do!